src/paths.ts

```diff
@@ -6,34 +6,95 @@ import * as path from 'path';
 import {decode} from 'urlencode';
 
 /** converts a server-side XDebug file URI to a local path for VS Code with respect to source root settings */
-export function convertDebuggerPathToClient(fileUri: string|url.Url, localSourceRoot?: string, serverSourceRoot?: string): string {
+export function convertDebuggerPathToClient(fileUri: string|url.Url, localSourceRoot?: string, serverSourceRoot?: string, pathMapping?: { [index: string]: string; }): string {
+
     if (typeof fileUri === 'string') {
         fileUri = url.parse(fileUri);
     }
+
     // convert the file URI to a path
     let serverPath = decode(fileUri.pathname!);
+
     // strip the trailing slash from Windows paths (indicated by a drive letter with a colon)
     const serverIsWindows = /^\/[a-zA-Z]:\//.test(serverPath);
+
     if (serverIsWindows) {
         serverPath = serverPath.substr(1);
     }
+
+    if ( typeof pathMapping !== 'undefined' ) {
```

Comment:
Maybe turn this on => [Whitespace](https://palantir.github.io/tslint/rules/whitespace/)